embedded_sass.proto

```diff
@@ -115,6 +115,12 @@ message InboundMessage {
 
     // Whether to encode the formatted message of errors and logs in ASCII.
     bool alert_ascii = 9;
+    
+    // Whether to report all deprecation warnings or only the first few ones.
```

Comment:
Remember that this is a specification of the compiler's behavior. Be more explicit and detailed about what the compiler is allowed to do, and how this flag affects it. For example: "If this is `false`, the compiler may choose not to send events for repeated deprecation warnings. If it's `true`, the compiler must emit an event for every deprecation warning it encounters."

Reply:
@nex3 updated